docker/mainnet/nodes/stacks/Config.toml

```diff
@@ -0,0 +1,23 @@
+[node]
+working_dir = "/stacks"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
+prometheus_bind = "0.0.0.0:9153"
+
+bootstrap_node = "02196f005965cebe6ddc3901b7b1cc1aa7a88f305bb8c5893456b8f9a605923893@seed.mainnet.hiro.so:20444,02539449ad94e6e6392d8c1deb2b4e61f80ae2a18964349bc14336d8b903c46a8c@cet.stacksnodes.org:20444,02ececc8ce79b8adf813f13a0255f8ae58d4357309ba0cedd523d9f1a306fcfb79@sgt.stacksnodes.org:20444,0303144ba518fe7a0fb56a8a7d488f950307a4330f146e1e1458fc63fb33defe96@est.stacksnodes.org:20444"
+
+[burnchain]
+chain = "bitcoin"
+mode = "mainnet"
+peer_host = "bitcoin"
+username = ""
+password = ""
+rpc_port = 8332
+peer_port = 8333
+
+[connection_options]
+private_neighbors = false
+
+[[events_observer]]
```

Comment:
Should we add here the sbtc-signer event observer?

Reply:
Yes it's in todo